taxonomies/ingredients.txt

```diff
@@ -46,7 +46,7 @@ stopwords:da:av
 stopwords:de:und, mit, von, aus, enthält, enthalten
 stopwords:el:περιέχει
 stopwords:en:contains, contain, from, with, produced with
-stopwords:es:contiene,de,del,la,el,las,los,con,y,minimo
+stopwords:es:contiene,de,del,la,el,las,los,con,y,minimo,maximo,procedente,agricultura,ecologic,ecologica,puede,contener
```

Comment:
```suggestion
stopwords:es:contiene,de,del,la,el,las,los,con,y,minimo,maximo
```

for the organic labels, we now recognize them in another way.   same for traces. There is some code in lib/ProductOpener/Ingredients.pm to do that, we can improve it for Spanish.

Reply:
ok, organic labels removed